### DISCUSSION TITLE
How can we add xsi:type in a tag

### DISCUSSION THREAD

**Discussion by devDzign** [phpro/soap-client#424] [Q&A]
```
<thirdParty **xsi:type="v1:physicalThirdParty"** xmlns:xsi="xxxxxxx">
    </thirdParty>

```
Hello,

I'd like to add  the meta-data xsi:type and xmlns:xsi in my thirdPary tag.
Could you please tell us if there a way to do so ?

Regards,

**Comment by veewee** [comment]
It depends... What does the content of the tag look like?
Is there a way from the generated classes?
Have you tried using type converters?
Final option would be to manipulate the XML by using an http Middleware.
